docs/webui_tutorial.rst

```diff
@@ -4,6 +4,9 @@ Web Application Tutorial
 .. toctree::
   :maxdepth: 2
 
+.. NOTE::
+      We are currently reworking our documentation for Raiden and just finished updating this tutorial for the web application. You can view the updated web application tutorial in this `new but incomplete version of the documentation <http://doc.raiden.network/>`_.
```

Comment:
I don't see anything on http://doc.raiden.network/

Reply:
Hmm, perhaps the DNS configuration hasn't gone through yet.

Reply:
Fyi, I did one more commit with text mentioning the Wizard and links to GitBook. However I am still waiting for the DNS configuration. I'll re-request a review as soon as that is working.

Reply:
GitBook links are live now and pointing to the correct sections.

Reply:
https://docs.raiden.network/ works for me, but not https://doc.raiden.network/

Reply:
I made a new commit a couple of minutes ago changing all links to the https://docs.raiden.network.

Reply:
Ah sorry.